SPRINTUSER_README.md

```diff
@@ -1,32 +1,94 @@
-  - Run install_ddev.sh via terminal (git-bash on Windows) from the directory where this SPRINTUSER_README.md is with `./install_ddev.sh` - On Windows also run the ddev_windows_installer bundled.
-4. Create Drupal 8 instance to use during sprint:
-  - cd to `~/sprint` and run `./start_sprint.sh`
-5. There should be a new sprint folder named `sprint-YYYYMMDD-HHMM` (with today's date and the time you ran `start_sprint`). Start the Drupal environment by using cd to move to the dated sprint folder, and run `./start_clean.sh`
-6. After start_clean is finished, it will provide you with a set of URLs and further instructions for using your sprint environment.
+## Prerequisites
+
+* A computer with 8gb memory or greater.
+* Windows 7 or higher, MacOS Sierra or higher or a recent/stable Linux distribution.
+* A robust code editor such as Visual Studio Code, Atom, PhpStorm, Netbeans, vim or emacs (this may be provided as part of this package).
+
+**⚠️ You can still install and contribute to Drupal even without these requirements using [Drupal's quick-start command](https://www.drupal.org/docs/8/install/quick-start-launch-a-local-demo-version-of-drupal-8-using-4-brief-steps)**!
+
+## Getting Started
+
+1. [Get The Files](#get-the-files)
+2. [Extract drupal_sprint_package](#extract-files)
+3. [Install Docker and Other Requirements](#install)
+4. [Open Terminal](#open-terminal)
+5. [Install Contribution Tools](#install-tools)
+
+<a name="get-the-files"></a>
+### 1. Get The Files
+
+* USB Drive - Copy drupal_sprint_package-RELEASE.zip and Docker installer for your Operating System from the USB drive to your Desktop.
+* ResilioSync - Find the folder/directory that was downloaded and copy its contents to your Desktop.
+* GitHub - Download drupal_sprint_package-RELEASE.zip from https://github.com/drud/quicksprint/releases.
+
+<a name="extract-files"></a>
+### 2. Extract drupal_sprint_package directory
+
+1. Extract the drupal_sprint_package-RELEASE.zip file to your Desktop. It is important that these files exist within your "user directory".
+2. Browse to this directory.
+
+<a name="install"></a>
+### 3. Install Docker and Other Requirements
+
+#### 3.1 Docker CE or Docker Toolbox
+
+Find the Docker installer for your Operating System underneath the **installs** directory. It is important to install the version of Docker provided for compatibility with the tools.
+
+ Operating System | Docker Version | Installer
+ ---------------- | -------------- | -----------------
+ Windows 10 Pro, Enterprise (HyperV enabled) | Docker CE | "Docker for Windows Installer.exe"
+ Windows 10 Home | Docker Toolbox | DockerToolbox-VERSION-ce.exe
+ Windows 7 (or no HyperV enabled)| Docker Toolbox | DockerToolbox-VERSION-ce.exe
+ MacOS | Docker CE | Docker.dmg
+ Linux | Docker CE, docker-compose | See [Linux instructions](https://docs.docker.com/install/#docker-ce)
+
+**⚠️ Docker Toolbox users**: Please install "Git for Windows" and "VirtualBox" as needed during the install process.
```

Comment:
I'm not sure, now that 2.18.0 is included, whether we should explicitly tell users to uncheck Git for Windows as I'm not sure if that will install Quickstart terminal. Reinstalling is fine. I tested that today.

Reply:
Oh, good point. Do this as it works. I forgot that it automatically includes that.